secretsmanager/src/openssl_pass.cpp

```diff
@@ -0,0 +1,46 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the
+// "License").  You may not use this product except in compliance with the
+// Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+//
+// This convenience header combines different block implementations.
+
+#include "openssl_pass.h"
+
+#include <string>
+#include <vector>
+
+#include <cryptopp/cryptlib.h>
+#include <cryptopp/sha.h>
+#include "misc.h"
+
+namespace concord::secretsmanager {
+
+KeyParams derive_key_pass(const std::string_view pass,
```

Comment:
nit: function names should follow `camelCase` pattern.

Reply:
My bad. I always mess up the code style :(